src/styles/blocks/communication.scss

```diff
@@ -0,0 +1,19 @@
+.communication {
+  &__title {
+    margin-bottom: 8px;
+    font-size: 14px;
+    line-height: 140%;
+    font-weight: 300;
+  }
+
+  &__information {
+    margin-bottom: 24px;
+    font-size: 16px;
+    line-height: 140%;
+    font-weight: 400;
+
+    &:last-child {
```

Comment:
better use

```
&__information:not(:last-child) {
  margin-bottom: 24px;
}
```

Reply:
also better use this approach to improve readability 

 `&:last-child {` ->  `&__information:last-child {`

Reply:
@mrkovaliv, look, I had a problem with the sticky menu. If I throw away the display none property - it stays on the screen and takes 100vh place(
I don't understand why it happening because in the lesson there was shown a possibility to exclude display manipulations.

Reply:
Could do you please ask this question in q&a chat (I think it'll be useful for other students)?